### PR TITLE
Exclude archived locations in reminders and alerts

### PR DESCRIPTION
**Pull request by kkrampa**
@gcapalbo

### CODE REVIEW

File: custom/ewsghana/alerts/ongoing_non_reporting.py [hunk 1]
```diff
@@ -12,7 +12,8 @@ class OnGoingNonReporting(WeeklyAlert):
     message = ONGOING_NON_REPORTING
 
     def get_sql_locations(self):
-        return SQLLocation.objects.filter(domain=self.domain, location_type__name='district')
+        return SQLLocation.objects.filter(domain=self.domain, location_type__name='district')\
```
Comment (czue) [inline custom/ewsghana/alerts/ongoing_non_reporting.py]: you can also use `SQLLocation.active_objects` for this. might be cleaner. https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/locations/models.py#L218

**Comment by kkrampa** [comment]
@czue Thanks! Changed.

File: custom/ewsghana/alerts/ongoing_stockouts.py [hunk 2]
```diff
@@ -61,4 +61,4 @@ def get_descendants(self, sql_location):
     def get_sql_locations(self):
         return SQLLocation.objects.filter(
             domain=self.domain, location_type__name__in=['country', 'region']
```
Comment (czue) [inline custom/ewsghana/alerts/ongoing_stockouts.py]: (did you mean to skip this one?)

**Comment by czue** [comment]
:+1: from me when tests pass

**Comment by kkrampa** [comment]
@czue It seems unrelated but maybe it's worth to run again.

**Comment by gcapalbo** [comment]
looks good, restarted build and just waiting for it to finish

**Comment by kkrampa** [comment]
:white_check_mark:

**Pull request merged**